### ISSUE TITLE
[TS] method returning a field compared by == with a String makes the TS fail

### ISSUE THREAD

**Issue by javierfernandes** [uqbar-project/wollok#1187]
This makes the TS fail (EValidator error)

```
class Color {
	const nombre
	constructor(n) { nombre = n }
	method esEstrambotico() { return nombre == "Violeta" }
}
```

This likes renders it unusable to tell the truth.

**Comment by javierfernandes** [comment]
same for comparing the parameter

```
class MochilaChica {
	method puedeCargar(peso) = peso <= 40
}
```

**Comment by npasserini** [comment]
First example has been included and tested in 7dc3fbe83c4077dc670ccf1d0889c6d4a3febc7d.
Will check for the second one before closing this.
